spec/models/member_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe Member, type: :model do
+               
+  before :each do
+    @first                      = "Steve"
+    @last                       = "Jobs"
```

Comment:
Remove the large space gap. Do the same for the other lines, e.g., line 6 to line 12, line 15 to line 21.